ui/src/shared/middleware/errors.js

```diff
@@ -35,36 +41,39 @@ const errorsMiddleware = store => next => action => {
         message ===
         `This organization is private. To gain access, you must be explicitly added by an administrator.` // eslint-disable-line quotes
```

Comment:
seems like this should also become factored out into a centralized notification message